server/conn.go

```diff
@@ -530,6 +532,17 @@ func (cc *clientConn) handleLoadData(loadDataInfo *executor.LoadDataInfo) error
 
 func (cc *clientConn) handleQuery(sql string) (err error) {
 	startTs := time.Now()
+	defer func() {
+		// Add metrics
+		t := time.Since(startTs)
+		queryHistgram.Observe(float64(t))
```

Comment:
no, you name the metric with `handle_query_duration_seconds `, so we must use Seconds here.
Btw, `Seconds()` doesn't return only seconds, it returns like 1.1111111 , see https://golang.org/pkg/time/#Duration.Seconds